server/container_create.go

```diff
@@ -393,6 +393,11 @@ func (s *Server) getAppArmorProfileName(annotations map[string]string, ctrName s
 	}
 
 	if profile == apparmor.ProfileRuntimeDefault {
```

Comment:
> Basically we should only call EnsureDefaultApparmorProfile if:
a. The user is using the default profile and hasn't set up their own; and
b. The container being started is using ProfileRuntimeDefault

I think this block maybe have already met your requirement(if change `apparmor.LoadDefaultAppArmorProfile()` to `apparmor. EnsureDefaultApparmorProfile`).